pkg/controller/configmap/configmap_controller.go

```diff
@@ -183,30 +279,32 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 
 	// Get network CRD configuration
 	networkConfig := &configv1.Network{}
-	err = r.client.Get(context.TODO(), types.NamespacedName{Name: operatortypes.NetworkCRDName}, networkConfig)
-	if err != nil {
-		if apierrors.IsNotFound(err) {
-			// Request object not found, could have been deleted after reconcile request.
-			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
-			// Return and don't requeue
-			log.Info("Cluster network CRD is not found")
-			r.status.SetDegraded(statusmanager.ClusterConfig, "NoClusterConfig", "Cluster network CRD is not found")
-			return reconcile.Result{}, nil
+	if r.sharedInfo.AdaptorName == "openshift4" {
```

Comment:
You can use a ReconcileConfigMapOc func to get network CR

Reply:
Done, thanks.